docs/internals/contributing/new-contributors.txt

```diff
@@ -27,39 +27,39 @@ Triage tickets
 If an `unreviewed ticket`_ reports a bug, try and reproduce it. If you can
 reproduce it and it seems valid, make a note that you confirmed the bug and
 accept the ticket. Make sure the ticket is filed under the correct component
-area. Consider writing a patch that adds a test for the bug's behavior, even if
-you don't fix the bug itself. See more at :ref:`how-can-i-help-with-triaging`
+area. Consider writing a test for the bug's behavior, even if you don't fix the
+bug itself. See more at :ref:`how-can-i-help-with-triaging`
 
-Review patches of accepted tickets
-----------------------------------
+Review pull requests of accepted tickets
```

Comment:
The term "pull request" may not be broad enough here. Majority of the older issues has their patches without a pull request and I still see people uploading patches to Trac from time to time.